### ISSUE TITLE
Change titles to articles: Developer Tutorials > Onboard REST APIs.....

### ISSUE THREAD

**Issue by JamesBauman** [zowe/docs-site#149]
In the Dev tutorials section of the docs-site, a suggestion was made by a participant during the playback on   October 15, 2018.
The suggestion is to shorten the names of the topics, they are currently too wordy, which makes the use case unclear.  

We are recommending the following changes to the article titles:

**From:** Onboard an existing Spring Boot REST API service with Zowe API Mediation Layer
**To:** Java REST APIs with Spring Boot  

**From:** Onboard an existing Java REST API service without Spring Boot with Zowe API Mediation Layer
**To:** Java REST APIs without Spring Boot 

**From:** Onboarding an existing Java Jersey REST API service
**To:** Java Jersey REST APIS 

**From:** Onboard an existing REST API service with the Zowe API Mediation Layer without code changes
**To:** REST APIs without code changes

**Note:**  For consistency, we are appending -ing to Onboard in the section heading Onboard REST APIs to the APIML. We apply the fix in the the `config.js` file, 

```
title: '**Onboard** REST APIs to the API Mediation Layer',
          collapsable: true,
          children: [
            'api-mediation-onboard-a-sprint-boot-rest-api-service', 
            'api-mediation-onboard-an-existing-java-rest-api-service-without-spring-boot-with-zowe-api-mediation-layer', 
            'api-mediation-onboard-an-existing-java-jersey-rest-api-service', 
            'api-mediation-onboard-an-existing-rest-api-service-without-code-changes']
```

**Comment by janan07** [comment]
I have discussed these title changes with our PO and, for the most part, we agree with the proposed titles.
Two changes that we suggest are:

1. A clearer title for the parent article is, "Onboard existing REST APIs with the API Mediation Layer"
2. A clearer title for the last title on the list is "REST APIs without code changes required"

As a general note, removing the word "Onboard" from the article titles also makes the purpose of each independent article much less clear as the purpose of the articles is dependant on the title of the parent article.

**Comment by JamesBauman** [comment]
Hi Andrew @janan07 -  As for the parent article title.... Please  use "Onboarding."  IF this was docops content, Onboard is OK.. However, we have an agreed style with IBM/Rocket to append -ing to all article titles that start with a verb. Also, the other parent articles in this section start with a gerund. so this is also a matter of consistency.  

thx,
jb

**Comment by janan07** [comment]
Thanks for the feedback, Jim. Sounds good.
